web/concrete/src/Editor/LinkAbstractor.php

```diff
@@ -35,11 +37,16 @@ public static function translateTo($text) {
 		$r = $dom->str_get_html($text);
 		if ($r) {
 			foreach($r->find('img') as $img) {
-				$src = $img->src;
-				$alt = $img->alt;
-				$style = $img->style;
-				if (preg_match($imgmatch, $src, $matches)) {
-					$img->outertext = '<concrete-picture fID="' . $matches[1] . '" alt="' . $alt . '" style="' . $style . '" />';
+
+				$attrString = "";
+				foreach($img->attr as $key => $val) {
+					if(!in_array($key, self::$blackListImgAttributes)) {
+						$attrString .= "$key='$val' ";
+					}
+				}
+
+				if (preg_match($imgmatch, $img->src, $matches)) {
+					$img->outertext = '<concrete-picture fID="' . $matches[1] . '" ' . $attrString . ' />';
```

Comment:
Note that, for an empty `attrString`, this results in two consecutive spaces. This is why the Travis build is failing.